cmd/nvidia-dra-plugin/device_state.go

```diff
@@ -390,6 +393,21 @@ func (s *DeviceState) applySharingConfig(ctx context.Context, config configapi.S
 		allocatableDevices[r.Device] = s.allocatable[r.Device]
 	}
 
+	// allow devices only with cuda compute compatility >= 7.5 as time slicing and MPS does not work with old arch
+	shareableAllocatableDevices := make(AllocatableDevices)
+	for device, deviceType := range allocatableDevices {
+		if deviceType.Gpu != nil {
+			cudaCCv := "v" + strings.TrimPrefix(deviceType.Gpu.cudaComputeCapability, "v")
+			gpuUUID := deviceType.Gpu.UUID
+			if semver.Compare(semver.Canonical(cudaCCv), semver.Canonical("v7.5")) >= 0 {
```

Comment:
@guptaNswati where does the `v7.5` threshold come from? In #58 we check for `>= v7.0` and for [MPS specifically](https://docs.nvidia.com/deploy/mps/#limitations), `v3.5` is mentioned.

Reply:
I picked it from our device-plugin code checking if its Volta https://github.com/NVIDIA/k8s-device-plugin/blob/main/cmd/mps-control-daemon/mps/device.go#L51